File: 2278-percentage-of-letter-in-string/2278-percentage-of-letter-in-string.c
int percentageLetter(char * s, char letter){
    int i = 0,perc = 0;
    
    while(s[i]){
        if(s[i] == letter) perc++;
        i++;
    }
    perc = (perc * 100) / i;   //i = length of s
    return perc;
}

File: 2278-percentage-of-letter-in-string/2278-percentage-of-letter-in-string.swift
class Solution {
    func percentageLetter(_ s: String, _ letter: Character) -> Int {
        let len = s.count
        var perc = 0
        
        for i in s {
            if i == letter {
                perc += 1
            }
        }
        perc = (perc * 100) / len
        return perc
    }
}